debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/LogMinerHelper.java

```diff
@@ -252,8 +254,24 @@ static Duration getTimeDifference(Connection connection) throws SQLException {
     static void startLogMining(Connection connection, Long startScn, Long endScn,
                                OracleConnectorConfig.LogMiningStrategy strategy, boolean isContinuousMining)
             throws SQLException {
+        LOGGER.trace("Starting log mining startScn={}, endScn={}, strategy={}, continuous={}", startScn, endScn, strategy, isContinuousMining);
         String statement = SqlUtils.startLogMinerStatement(startScn, endScn, strategy, isContinuousMining);
         executeCallableStatement(connection, statement);
+        try {
+            executeCallableStatement(connection, statement);
+        }
+        catch (SQLException e) {
+            // Check if we got ORA-01291 or ORA-01284
+            if (e.getErrorCode() == 1291) {
+                try {
+                    logLogMinerLogEntries(connection);
+                }
+                catch (SQLException e2) {
+                    LOGGER.error("Failed to capture logminer log entries", e2);
```

Comment:
Casing: is it "LogMiner"? Or even "Log Miner"? Worth double-checking.

Reply:
We have likely been a bit inconsistent here and normally we've used `Logminer` or `logminer` but I'll confirm and adjust accordingly.

Reply:
Excellent, thx!

Reply:
It would seem the Oracle documentation uses `LogMiner`.  While its a tad unrelated to this PR I'll add a small commit to it to standardize it in the codebase.